include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -514,15 +514,24 @@ struct __scan
 
             // 1. reduce
             auto __k = 1;
+            auto __next_k = 2;
```

Comment:
What the `auto` is here? Let's mark explicitly. Seems it has the same type as `__wgroup_size`